skbeam/core/accumulators/tests/test_binned_statistic.py

```diff
@@ -23,51 +24,112 @@ def setup(self):
         colarr = np.arange(colsize)
         rowgrid, colgrid = np.meshgrid(rowarr, colarr, indexing='ij')
         self.rgrid = np.sqrt(rowgrid**2 + colgrid**2)
+        self.phigrid = np.arctan2(colgrid, rowgrid)
```

Comment:
Hmm.  I'll have to think about this.  That suggests that using the radial_grid/phi_grid is fundamentally changing the grid compared to what I had previously.  Ideally I would like to use your calls, but keep the grid identical to what it was previously so that the definition of phi/r don't change, for example.  But perhaps I'm missing something, so feel free to disagree...